### PR TITLE
Add aspects to curriculum/javascript/core/strings-ii

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/strings-ii/most-efficient-way-to-build-html-strings.md?ref=ae6f5c03f7a79340a0a33f932d13f0bdd776b822

File: javascript/core/strings-ii/js-practice-string-charat.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ standards:
   javascript.standard-library.0: 1000
   javascript.write-expressions.0: 1000
 
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/strings-ii/js-practice-string-charat.md]: I would add `introduction`

File: javascript/core/strings-ii/js-practice-string-concat.md [hunk 2]
```diff
@@ -16,6 +16,9 @@ standards:
   javascript.standard-library.0: 1000
   javascript.write-expressions.0: 1000
 
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/strings-ii/js-practice-string-concat.md]: Definitely `introduction` as well

File: javascript/core/strings-ii/js-practice-string-indexof.md [hunk 3]
```diff
@@ -16,6 +16,9 @@ standards:
   javascript.standard-library.0: 1000
   javascript.write-expressions.0: 1000
 
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/strings-ii/js-practice-string-indexof.md]: `introduction`

File: javascript/core/strings-ii/js-practice-string-slice.md [hunk 4]
```diff
@@ -11,6 +11,10 @@ type: exercise
 
 link: http://www.codewars.com/kata/57274562c8dcebe77e001012
 linkType: codewars
+
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/strings-ii/js-practice-string-slice.md]: `introduction`

File: javascript/core/strings-ii/js-practice-string-touppercase.md [hunk 5]
```diff
@@ -11,6 +11,10 @@ type: exercise
 
 link: http://www.codewars.com/kata/5728203b7fc662a4c4000ef3
 linkType: codewars
+
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/strings-ii/js-practice-string-touppercase.md]: `introduction`

File: javascript/core/strings-ii/js-practice-string-trim.md [hunk 6]
```diff
@@ -11,6 +11,10 @@ type: exercise
 
 link: http://www.codewars.com/kata/5729b103dd8bac11a900119e
 linkType: codewars
+
+aspects:
+  - workout
```
Comment (ryanwarsaw) [inline javascript/core/strings-ii/js-practice-string-trim.md]: `introduction`

**Pull request merged**